lib/tasks/prod_seed.rake

```diff
@@ -0,0 +1,548 @@
+namespace :db do
+  desc "Loading Rails World 2024 conference data."
+  task :prod_seed, [:month_number, :day_number] => :environment do |t, args|
+    month = args[:month_number].present? ? args[:month_number].to_i : 9
+    start_day = args[:day_number].present? ? args[:day_number].to_i : 26
+    second_day = start_day + 1
+    image_path = "app/assets/images/speakers"
+
+    # Create Conference
+    conference = Conference.find_or_create_by!(name: "Rails World 2024")
+
+    # Create Locations
+    track_1 = conference.locations.find_or_create_by!(name: "Track 1")
+    track_2 = conference.locations.find_or_create_by!(name: "Track 2")
+    lightning_track = conference.locations.find_or_create_by!(name: "Lightning Track")
+
+    # Create Tags
+    # No tags for now
+
+    # Create Speakers
+    matz_speaker = Speaker.find_or_create_by!(name: "Yukihiro \"Matz\" Matsumoto") do |speaker|
```

Comment:
I forgot that `name` is coming from the profile. I'm getting this error:
```
ActiveRecord::StatementInvalid: SQLite3::SQLException: no such column: speakers.name (ActiveRecord::StatementInvalid)
```

Consider using something like this:
```
    matz_speaker = Profile.find_by(name: "Yukihiro \"Matz\" Matsumoto", profileable: "Speaker")&.speaker
    matz_speaker ||= Speaker.create!(
      name: "Yukihiro \"Matz\" Matsumoto",
      job_title: "Creator of Ruby",
      bio: "Matz is the creator and chief designer of the Ruby programming language.",
      github_url: "https://github.com/matz",
      linkedin_url: "",
      twitter_url: "https://x.com/yukihiro_matz",
      image: Rails.root.join("#{image_path}/m-matsumoto.jpg").open
    )
```